vpd-tool/include/vpd_tool.hpp

```diff
@@ -39,5 +62,20 @@ class VpdTool
                     const std::string& i_recordName,
                     const std::string& i_keywordName, const bool i_onHardware,
                     const std::string& i_fileToSave = {});
+
+    /**
+     * @brief Dump the given inventory object in JSON format to console.
+     *
+     * For a given Object Path of a FRU, this API dumps the following properties
+     * of the FRU in JSON format to console:
+     * - Present property, Pretty Name, Location Code, Sub Model
+     * - SN, PN, CC, FN, DR keywords under VINI record.
+     * - type and TYPE of the FRU
```

Comment:
what is type and TYPE of the FRU?

Reply:
From the old `vpd-tool` , 

1. "type" property specifies the `extraInterfaces` property which is `null`.Eg. `xyz.openbmc_project.Inventory.Item.Cpu`
2. "TYPE" tag default value is "FRU", unless there is a "type" tag in the System Config JSON. As per new requirement, since `vpd-tool` doesn't have System Config JSON, I think we may have to skip dumping this "TYPE" tag.